src/lib/form/useOnSubmit.ts

```diff
@@ -53,6 +55,9 @@ export const useOnSubmitEdit = <TFormValues extends IdentifiableObject>({
                 message: i18n.t('Saved successfully'),
                 success: true,
             })
+            queryClient.invalidateQueries({
```

Comment:
I think this makes a lot of sense, however I'm not sure if we want to couple the sectionPath to the resource? 
I think it's more correct to use `section.namePlural` if we want to do this. However, conceptually the idea is that sections are not directly coupled to API-resources - thats what the schemas are for. 

So following the concept in my head we would need the schema, and do `schema.plural` here?

Initially I was thinking to add `refetchOnMount: true` on the queries for the list. That should fix the problem in this case. But I do agree that invalidating the query makes a lot of sense as well.

Reply:
Sounds good, i'll chande it to namePlural and keep it as it is then (i do like being explicit about it)